src/main/java/org/radarcns/redcap/config/ManagementPortalInfo.java

```diff
@@ -27,10 +27,14 @@
      * Management Portal project identifier. As long as the Management Portal is centralised, this
```

Comment:
Is there ever any senario where the management portal is not centralised?

Reply:
No i guess. If someone is using another instance of MP then it they will also deploy an instance of redcap-integration with it. It makes no sense to use another server's instance of redcap-integration.